src/components/jury.jsx

```diff
@@ -0,0 +1,121 @@
+        {"avatar": AlexCiteaAvatar, "name": "Alex Vasile Iosif Citea", "occupation": "Malware Analyst @ Bitdefender"},
+        {"avatar": AlexComanAvatar, "name": "Alexandru Coman", "occupation": "Cloud Engineer  @ Cloudbase"},
+        {"avatar": VladAvatar, "name": "Vlad Temian", "occupation": "Software Developer @ Presslabs"},
+        {"avatar": DefaultAvatar, "name": "Alexandru Buicescu", "occupation": "Inginer Software @ Indycoding"}
+    ];
+    var web = [
+        {"avatar": DefaultAvatar, "name": "Narcisa Dima", "occupation": "Profesor @ Liceul 'Ion Barbu' Pitesti"},
+        {"avatar": DefaultAvatar, "name": "Istrate Nicolae", "occupation": "Profesor @ Colegiul National 'Ienachita Vacarescu' din Targoviste"},
+        {"avatar": AndreiAAvatar, "name": "Andrei Adoamnei", "occupation": "Head of Development @ Conversion Network LLC"},
+        {"avatar": RobertAvatar, "name": "Robert Dolca", "occupation": "Inginer Software @ Intel"},
+        {"avatar": AlexPAvatar, "name": "Alex Palcuie", "occupation": "Referent @ Cancelaria Prim-Ministrului"},
+        {"avatar": SabinMarcuAvatar, "name": "Sabin Marcu", "occupation": "student @ University of Southampton"}
+    ];
+
+    var criteria = [
+      {"name": "Educațional", "link": "https://docs.google.com/document/d/10ygW8gpijQJE94CbMMNxhrt_ju3X-tMjPATf7WcjTW8/pub"},
+      {"name": "Multimedia", "link": "https://docs.google.com/document/d/12UY9ByKQt7CpBj11vwg1HPrsKOH-uTL9kFCEdxBgvAY/pub"},
+      {"name": "Mobile", "link": "https://docs.google.com/document/d/1jqwoAduZoHjnj01Q6eQE0DWNB-DeMQouLumkW753mUA/pub"},
+      {"name": "Roboți", "link": "https://docs.google.com/document/d/1n0yoqA3n9heBiYWgmeg-lfXYPvSMVMRhGZrHRYKhDVk/pub"},
+      {"name": "Utilitar", "link": "https://docs.google.com/document/d/19HTucZgQWY92nZ-NDoKRWlE63hipb_IkZVcMxHyOCdc/pub"},
+      {"name": "Web", "link": "https://docs.google.com/document/d/1ZgdZz_5JHJSZyxWnZWzP162NZ2H3CkBojyp5p6cKNDI/pub"}
+    ];
+
+    return <div className="jury">
+        <div className="blue-section-wrapper">
+          <div className="container">
+            <div className="row blue-section">
+              <Header isLoggedIn={this.props.isLoggedIn} />
+              <h1>Juriul InfoEduca&#355;ie</h1>
+              <h2>Membrii juriului pe categorii</h2>
+            </div>
+          </div>
+        </div>
+
+        <JuryDescription iconClass="educational" name="educațional" members={educational}/>
+        <JuryDescription iconClass="media" name="multimedia" members={media}/>
+        <JuryDescription iconClass="robots" name="roboți" members={robots}/>
+        <JuryDescription iconClass="utility" name="utilitar" members={utilitar}/>
+        <JuryDescription iconClass="web" name="web" members={web}/>
+
+        <div className="orange-section-wrapper">
+          <div className="container">
+            <Row className="orange-section">
+              <Row className="jury-criteria-desc">
```

Comment:
Replace this with `<Col md={12}>`. The nesting [should](http://react-bootstrap.github.io/components.html#grids) be `container > row > column > content`.